### PR TITLE
fix: ReferenceError 'window is not defined' in Seasons-component

### LINKED ISSUE #94: Rewrite seasons component with updated look and feel
### Is your feature request related to a problem? Please describe.

Right now you have to click the season number to be able to see the switch. It would be easier if you could just swipe on the given card to go to the next season on mobile.

### Describe the solution you'd like

I would like the look and functionality of the seasons component to be less janky
I would like the season cards to be swipeable on mobile.

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Pankai222**
# Description

Can't use window on the server :D

Fixes #94 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

File: frontend/src/lib/components/details/seasons.svelte [hunk 1]
```diff
@@ -6,10 +6,17 @@
...
   import "swiper/css/free-mode"
   import "swiper/css/navigation"
+  import { onMount } from "svelte"
 
   export let seasons
 
-  const INITIAL_OVERVIEW_LENGTH: number = window.visualViewport.width > 640 ? 170 : 100
+  let viewPortWidth: number
+
+  const INITIAL_OVERVIEW_LENGTH: number = viewPortWidth > 640 ? 170 : 100
```
Comment (AndreasPB) [inline frontend/src/lib/components/details/seasons.svelte:15]: `viewPortWidth` will always be `undefined` here, so `INITIAL_OVERVIEW_LENGTH` is always `100`. The `viewPortWidth`onMount()-mutation happens after the browser has loaded the content

What is the reason we need to chose between 170 and 100?

**Review by AndreasPB — CHANGES_REQUESTED** [review]
I'm not sure if other people have it, but I get this line through the Swiper thing 
image

File: frontend/src/lib/components/details/seasons.svelte (hunk 1 above)
Comment (AndreasPB) [reply frontend/src/lib/components/details/seasons.svelte:15]: If it's needed, you could do something like this, and get rid of the the JavaScript
```svelte
<script lang="ts">
  import { browser } from "$app/environment"
</script>
...whatever code
              <div class="pt-2 text-neutral-content">
                {#if browser && season.overview}
                  <ReadMore
                    currentDescriptionLength={window.visualViewport.width > 640 ? 170 : 100}
                    mediaDescription={season.overview}
                    initialDescriptionLength={window.visualViewport.width > 640 ? 170 : 100}
                  />
                {:else}
                  No season overview available.
                {/if}
...whatever code

Comment (Pankai222) [reply frontend/src/lib/components/details/seasons.svelte]: Wow, I feel stupid now xd 
As you mentioned, the switch between 170 and 100 is actually not needed. I think it was some leftover code I forgot to change.

**Review by AndreasPB — APPROVED** [review]

**Pull request merged**